book/src/Example.md

```diff
@@ -136,7 +136,11 @@ csi-hostpathplugin-9rp7c     2/2     Running   0          5s
 
 ### Deploy livenessprobe with CSI plugin
 
-Below is an example configuration which needs to be added to CSI driver/provisioner yaml:
+The CSI community provides a livenessprobe side-container that can be integrated with the CSI driver services (Node and Controller) to provide the liveness of the CSI service containers.
+
+The livenessprobe side-container will enable the integration with kubernetes liveness Probe, and will expose the endpoint on a httpGet request for CSI driver services. The same port is configured as `health-port` for liveness-probe side-container.
```

Comment:
Maybe reword to something like "The livenessprobe sidecar container will expose an http endpoint that be used in a Kubernetes liveness probe."

Reply:
done